recipes/r-bit/meta.yaml

```diff
@@ -80,8 +80,8 @@ about:
 # Title: A class for vectors of 1-bit booleans
 # Version: 1.1-12
 # Date: 2012-01-17
-# Author: Jens Oehlschlägel <Jens.Oehlschlaegel@truecluster.com>
-# Maintainer: Jens Oehlschlägel <Jens.Oehlschlaegel@truecluster.com>
+# Author: Jens Oehlschlaegel <Jens.Oehlschlaegel@truecluster.com>
```

Comment:
These are actually existing recipes that, when parsed by conda_build.metadata, raise UnicodeDecodeErrors. They are parsed when filtering recipes. So the options are:

- fix conda-build
- mangle people's names
- add them to the blacklist

Which do you prefer?